repair/repair.cc

```diff
@@ -1014,6 +1015,14 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
     auto id = repair_tracker().next_repair_command();
     rlogger.info("repair[{}]: starting user-requested repair for keyspace {}, repair id {}, options {}", id.uuid, keyspace, id.id, options_map);
 
+    // TODO Deexie add type and finish task. 
+    auto task = seastar::make_shared<repair_task>(task_id{id.uuid});
+    task->set_number(id.id);
```

Comment:
Hmm, maybe as a follow up patch at some point we can switch to task_manager for being the primary interface for dispatching task id's and sequence numbers (per module), rather than the repair_tracker.
But this can be done later.

In any case, it might be simpler to provide the id, keyspace, table, and possibly entity too as constructor args (with the strings defaulted to empty strings), rather than setting them functionally, one by one.